javaagent-extension-api/src/test/groovy/io/opentelemetry/javaagent/extension/instrumentation/InstrumentationModuleTest.groovy

```diff
@@ -11,11 +11,6 @@ import spock.lang.Specification
 
 class InstrumentationModuleTest extends Specification {
 
-  def setup() {
-    assert System.getenv().findAll { it.key.startsWith("OTEL_") }.isEmpty()
```

Comment:
Couldn't figure out what these were here for

Reply:
Probably to make sure that we don't override default conf that we test below

Reply:
Yeah I thought it could be a reason - I'm comfortable with deleting since I don't think we do this in other tests